### DISCUSSION TITLE
Missing repos from organization

### DISCUSSION THREAD

**Discussion by filiphsps** [renovatebot/renovate#22492] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

We’re missing three repos in on the dashboard that I can’t get to appear even with full re-installs.
It’s for the org @sweet-side-of-sweden and the repos are:
- https://github.com/sweet-side-of-sweden/sweetsideofsweden-frontend (private)
- https://github.com/sweet-side-of-sweden/react-tender - (public)
- https://github.com/sweet-side-of-sweden/react-shopify-analytics - (public)

The stopped working after being either transferred from another github account or renamed.
Would it be possible to do a complete reset so we can get it working agin?

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
Hi, could you confirm:
- Is this a recent install, or something older?
- Have you installed with "All repos" or "Selected repos"?
- When you say "the dashboard" are you referring to https://developer.mend.io?
- Are these the _only_ repos you have installed, or are some working and some not?

Cc @nabeelsaabna

**Reply by filiphsps** [reply]
Hi! 

Yes, this is a recent-ish install for all repos except one I believe.
I have tried installing with both all repos and specifically selecting all of the applicable ones.
Yep, that’s the dash I’m referring to.
No, there’s 3 that are working at them moment (according to the dashboard), but all of them were created directly in the org and have never been renamed unlike the ones that aren’t working.

// Filiph

**Reply by rarkins** [reply]
I just noticed this onboarding PR popped up a few minutes ago: https://github.com/sweet-side-of-sweden/opensource.sweetsideofsweden.com/pull/1

It's not in your list - was it one that you considered working?

Looking through your list manually in our DB I observe:

[sweet-side-of-sweden/sweetsideofsweden-frontend](https://github.com/sweet-side-of-sweden/sweetsideofsweden-frontend) (private) - it's enabled=false in our DB, meaning we think it's disabled

[sweet-side-of-sweden/react-tender](https://github.com/sweet-side-of-sweden/react-tender) - (public) - This one has never been seen

[sweet-side-of-sweden/react-shopify-analytics](https://github.com/sweet-side-of-sweden/react-shopify-analytics) - (public) - Also never been seen

Any chance you can tell me the old repository names if any of the above were renamed within the org?

**Reply by rarkins** [reply]
Hi, I've been able to locate some log messages in our backend which confirm there's a problem due to renaming (conflicts in row insertion, etc). It's unlikely something you can fix and will need one of our developers to address. I'll update you here once I have news

**Reply by filiphsps** [reply]
> I just noticed this onboarding PR popped up a few minutes ago: [sweet-side-of-sweden/opensource.sweetsideofsweden.com#1](https://github.com/sweet-side-of-sweden/opensource.sweetsideofsweden.com/pull/1)
> 
> It's not in your list - was it one that you considered working?
> 
> Looking through your list manually in our DB I observe:
> 
> [sweet-side-of-sweden/sweetsideofsweden-frontend](https://github.com/sweet-side-of-sweden/sweetsideofsweden-frontend) (private) - it's enabled=false in our DB, meaning we think it's disabled
> 
> [sweet-side-of-sweden/react-tender](https://github.com/sweet-side-of-sweden/react-tender) - (public) - This one has never been seen
> 
> [sweet-side-of-sweden/react-shopify-analytics](https://github.com/sweet-side-of-sweden/react-shopify-analytics) - (public) - Also never been seen
> 
> Any chance you can tell me the old repository names if any of the above were renamed within the org?

Yeah that repo is one of the entirely new ones. sweetsideofsweden-frontend and react-* are the ones I’m concerned with getting working. `sweetsideofsweden-frontend` was accidentally renamed to `sweetsidebysweden-frontend` for an hour or two but before that it was named `candybysweden-frontend`.

`react-tender` & `react-shopify-analytics` used to be under my private GitHub account before I transferred them a few hours ago so I’m assuming that’s why those two broke.

**Reply by filiphsps** [reply]
> Hi, I've been able to locate some log messages in our backend which confirm there's a problem due to renaming (conflicts in row insertion, etc). It's unlikely something you can fix and will need one of our developers to address. I'll update you here once I have news

Ah, Yeah I’d have to give em a new ID to resolve that then which would basically be creating entirely new repos. It’s not that big of a rush. Thanks for escalating it!

**Comment by gberche-orange** [comment]
Experiencing same problem after transferring a repo from a github personal account (https://github.com/gberche-orange/renovate-sample) to an organization account (https://github.com/orange-cloudfoundry/renovate-sample): the repo disappeared from the source personal account but is not appearing after 15+ minutes to the target organization account

**Comment by filiphsps** [comment, marked as answer]
This has now been resolved. Not sure if a developer manually fixed it or it was some sort of automation. If anybody else runs into this problem I'd recommend opening a new issue :)
